src/main/scala/com/kinja/amqp/AmqpConsumer.scala

```diff
@@ -25,6 +25,15 @@ class AmqpConsumer(
 	logger: Slf4jLogger
 )(val params: QueueWithRelatedParameters) extends AmqpConsumerInterface {
 
+	val actorName = "consumer_" + params.queueParams.name
+
+	/**
+	 * @inheritdoc
+	 */
+	override def cancel(): Unit = {
+		actorSystem.actorSelection(connection.path / actorName) ! CancelConsumer(actorName)
```

Comment:
This should invoke https://github.com/gawkermedia/amqp-client/blob/scala2.10_kinja/src/main/scala/com/github.sstone/amqp/Consumer.scala#L90